Kinvey/KinveyApp/MICAuthorizationGrantViewController.swift

```diff
@@ -19,10 +19,14 @@ class MICAuthorizationGrantViewController: UIViewController {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
         
-        Kinvey.sharedClient.initialize(
-            appKey: "kid_rJVLE1Z5",
-            appSecret: "cd385840cbd94e2caaa8f824c2ff7f46"
-        )
+        if let appKey = ProcessInfo.processInfo.environment["KINVEY_MIC_APP_KEY"],
```

Comment:
This is great! One suggestion - can you add instructions to the README for someone who is testing this locally vs on Travis?